Frameworks/AddressBook/ABAddressBookManagerInternal.mm

```diff
@@ -18,19 +18,52 @@
 #import "ABContactInternal.h"
 #import "UWP/WindowsApplicationModelContacts.h"
 
+@interface __ABContactOperation : NSObject
+
+@property BOOL shouldDelete;
+@property WACContact* contact;
+
+- (id)initWithContact:(WACContact*)contact shouldDelete:(BOOL)shouldDelete;
+
+@end
+
+@implementation __ABContactOperation
+
+- (id)initWithContact:(WACContact*)contact shouldDelete:(BOOL)shouldDelete {
+    if (self = [super init]) {
+        self.shouldDelete = shouldDelete;
+        self.contact = contact;
+    }
+
+    return self;
+}
+
+@end
+
 @interface _ABAddressBookManager ()
 
+- (void)_setError:(CFErrorRef*)error message:(NSString*)message;
```

Comment:
It is a private method that was designed to only be used internally to this class (the rest of the methods are called from ABAddressBook.mm and such, but don't need to call this one)

---
In reply to: [76674447](https://github.com/Microsoft/WinObjC/pull/935#discussion_r76674447) [](ancestors = 76674447)